ceph/ceph_admin/helper.py

```diff
@@ -564,6 +564,27 @@ def get_hosts_deployed(cls):
     return hosts
 
 
+def create_cephadm_pub_key_file(cls):
```

Comment:
IMO, we should keep this method in the test module.

Reply:
Moved the method to test module